esp/scm/ws_dfu.ecm

```diff
@@ -160,6 +174,8 @@ ESPStruct [nil_remove] DFUFileDetail
     [min_ver("1.37"), json_inline(1)] string jsonInfo;
     [min_ver("1.37")] binary binInfo;
     [min_ver("1.38")] string PackageID;
+    [min_ver("1.39")] ESPstruct DFUFilePartition Partition;
+    [min_ver("1.39")] ESPstruct DFUFileBloom Bloom;
```

Comment:
There can only be one PARTITION on an index but there can be several BLOOM attributes.